pennylane/ops/qubit/state_preparation.py

```diff
@@ -20,6 +20,8 @@
 from pennylane.operation import AnyWires, Operation
 from pennylane.templates.state_preparations import BasisStatePreparation, MottonenStatePreparation
 
+state_prep_ops = {"BasisState", "QubitStateVector", "QubitDensityMatrix"}
```

Comment:
What is this for I wonder?

Reply:
That was for `insert_ops`. Given the insight we can instead use `tape._prep`, this will be removed.